rewards/classes/TreeManager.py

```diff
@@ -204,9 +130,9 @@ def propose_root(self, rewards: dict) -> str:
                     url=self.discord_url,
                 )
         except Exception as e:
```

Comment:
should we raise an exception here after we log so k8s auto retries? maybe for another pr that's less busy